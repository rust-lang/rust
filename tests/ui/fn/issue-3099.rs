fn a(x: String) -> String {
    format!("First function with {}", x)
}

fn a(x: String, y: String) -> String { //~ ERROR the name `a` is defined multiple times
    format!("Second function with {} and {}", x, y)
}

fn main() {
    println!("Result: ");
}
