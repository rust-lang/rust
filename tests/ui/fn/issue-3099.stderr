error[E0428]: the name `a` is defined multiple times
  --> $DIR/issue-3099.rs:5:1
   |
LL | fn a(x: String) -> String {
   | ------------------------- previous definition of the value `a` here
...
LL | fn a(x: String, y: String) -> String {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `a` redefined here
   |
   = note: `a` must be defined only once in the value namespace of this module

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0428`.
